src/CppBridge.js

```diff
@@ -62,7 +62,7 @@ class CppBridge {
       if (options.destinations.length !== 1) {
         throw Error('Invalid number of destinations must be 1')
       }
-      if (options.destinations[0].send_amount !== 0) {
+      if (options.destinations[0].send_amount !== '0') {
```

Comment:
Since this file is copied from `@mymonero/mymonero-monero-client v2.0.0`, the bug must also exists in the upstream MyMonero code. Therefore, we need to also submit a patch to upstream (assuming the bug is still there), and add a comment to the very top of this file describing what we changed. Otherwise, if we ever re-sync with upsteam, we will lose your work.